letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -146,6 +146,17 @@ elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-joker" ]; then
     bashio::config.require 'dns.joker_password'
     PROVIDER_ARGUMENTS+=("--authenticator" "${DNS_PROVIDER}" "--${DNS_PROVIDER}-credentials" "/data/dnsapikey" "--${DNS_PROVIDER}-propagation-seconds" "${PROPAGATION_SECONDS}")
 
+# Plesk
+elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-plesk" ]; then
+    bashio::config.require 'dns.plesk_username'
+    bashio::config.require 'dns.plesk_password'
+    bashio::config.require 'dns.plesk_api_url'
+    if (( PROPAGATION_SECONDS < 120 )); then
+        bashio::log.info "Increasing DNS propagation limit for Plesk to at least 120 seconds."
```

Comment:
Where is this exactly comming from, is there anything specific to Plesk which makes this necessary? Or is this just by your observation on your end?

The PyPI example (at https://pypi.org/project/certbot-dns-plesk/) actually uses a default of 30s. Granted this might come from just a copied README.md. But there is no explicit mention that a higher value is necessary for Plesk.

Reply:
It was on my specific hosting-provider. Doing the default 60 seconds failed consistently. I forced 120 seconds for a better out of the box experience. Better remove it?

Reply:
For consistency, yes please remove it so it is the same as other dns providers. Also update the docs. You can mention it there explicitly, that is fine by me.

Reply:
@agners FTR there's the same exception for the TransIP provider, where it's bumped to 240s. If it really doesn't work reliably with lower numbers, maybe we can ignore the consistency argument here too.

Reply:
For TransIP 240 is also documented in the plug-in (see https://github.com/hsmade/certbot-dns-transip/blob/master/README.rst).

In the end, I don't care much. I was mostly wondering where the 120 came from. I am also fine leaving it as is. I guess most people will automate the run, and it really doesn't matter all that much if this runs 30s or 120s :sweat_smile: